package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-pattern-library",
-  "version": "0.10.0",
+  "version": "0.10.4",
```

Comment:
Good question. We haven't been able to update the doc site for a while, so we are now up to 0.10.4. See the CHANGELOG above for details. In particular, npm was down for a period and when it came up I was having publishing issues, so I created 0.10.4 as a noop just to try to get the version out.